src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1224,6 +1237,13 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
                             OutputProcessor::SOVTimeStepType::System,
                             OutputProcessor::SOVStoreType::Average,
                             this->name);
+        SetupOutputVariable(state,
+                            "Heat Pump Source Side Specific Heat",
+                            OutputProcessor::Unit::J_kgK,
+                            this->sourceSideCp,
+                            OutputProcessor::SOVTimeStepType::System,
+                            OutputProcessor::SOVStoreType::Average,
+                            this->name);
```

Comment:
I'll grab the fixed Cp calculation and also this Cp report variable in a standalone PR.

Reply:
And once again nm, the source cp was fixed already as well.  I'll include the new output variable at least... :)